src/main.cc

```diff
@@ -2,20 +2,64 @@
 
 #include <iostream>
 
+#include "landscape.h"
+#include "jpeg-texture.h"
+#include "sphere.h"
 #include "floor.h"
 
 namespace zennist {
 
 class Application final : public zukou::IExpansiveDelegate
 {
+  glm::vec4 rgbColor(float r, float g, float b)
```

Comment:
According to our name convention,
```suggestion
  glm::vec4 RgbColor(float r, float g, float b)
```

And this function seems better to be placed outside of `Application` class

Reply:
Thanks. Will move it outside.